frappe/public/js/frappe/dom.js

```diff
@@ -138,28 +138,29 @@ frappe.dom = {
 						return false;
 					}
 				})
-				.appendTo("#body");
-
+				.appendTo($('.modal:visible')); 
```

Comment:
Wont't this break it in every other case where modal isn't present on screen? :thinking: 

What am I missing.